packages/components/src/Tabs.stories.tsx

```diff
@@ -3,21 +3,109 @@ import * as React from "react";
 import { Tabs } from "./Tabs";
 import { Tab, TabComponentProps } from "./Tab";
 import styled from "styled-components";
+import { colors, fonts } from "./styleConstants";
 
-const StyledLi = styled.li`
-  background-color: whitesmoke;
-  padding: 3px 0 18px;
-  margin-bottom: 0;
-  width: 75px;
-  box-sizing: border-box;
-  text-align: center;
+const ListingsNavTabs = styled.div`
+  background-color: ${colors.accent.CIVIL_GRAY_4};
+  height: 76px;
+  margin: 0 auto 50px;
+  width: 100%;
+  & > ul {
+    justify-content: center;
+  }
+`;
+
+const ListingsNavTab = styled.li`
+  border-bottom: ${(props: TabComponentProps) =>
+    props.isActive ? "8px solid " + colors.accent.CIVIL_BLUE : "8px solid transparent"};
+  color: ${(props: TabComponentProps) => (props.isActive ? colors.accent.CIVIL_BLUE : colors.accent.CIVIL_GRAY_2)};
+  cursor: pointer;
+  font-family: ${fonts.SANS_SERIF};
+  font-size: 19px;
+  font-weight: 800;
+  margin: 39px 40px 0 0;
+  padding: 0 0 10px;
+  &:last-of-type {
+    margin: 39px 0 0 0;
+  }
+  &:hover {
+    border-bottom: ${(props: TabComponentProps) =>
+      props.isActive ? "8px solid " + colors.accent.CIVIL_BLUE : "8px solid " + colors.accent.CIVIL_GRAY_2};
+  }
+`;
+
+const ListingsSubnavTabs = styled.div`
+  margin: 30px auto 50px;
+  width: 100%;
+`;
+
+const ListingsSubnavTab = styled.li`
+  background-color: ${(props: TabComponentProps) => (props.isActive ? "#e9eeff" : "transparent")};
```

Comment:
Is `#e9eeff` in our `colors` constant? If not, can we use something close to it instead?

Reply:
Actually, this is just in Storybook, right?

Then this is probably ok.